### PR TITLE
[DE] Add `Help centre/Upgrading to lazer`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
- [x] Apply #8548 
- [x] Apply #8576 

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/Upgrading_to_lazer/de.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+
+Für Scores werden die Performance-Punkte bereits berechnet (du kannst das in der Sektion "Neuliche Plays" deines Profils erkennen), aber sie tragen noch nicht zum Gesamtwert bei.
+
+#### Verwendet lazer ScoreV2?
+
+Lazer verwendet aktuell eine neue, experimentelle Score-Implementierung, die ähnlich zu ScoreV2 ist, aber nicht die gleiche. Diese Funktion befindet sich noch in der Entwicklung und wir sind auf der Suche nach weiteren Rückmeldungen darüber, wie sie sich in verschiedenen Kontexten anfühlt (Einzelspielermodus, Ranglisten, Turniere etc.).
+
+<!-- lint ignore no-heading-punctuation -->
+
+#### Ich bevorzuge die klassische Spielstandanzeige, bei dem Scores richtig groß werden.
```
Comment (Phippe) [inline wiki/Help_centre/Upgrading_to_lazer/de.md:152]:
```suggestion
#### Ich bevorzuge die klassische Spielstandanzeige, bei der Scores richtig groß werden.
```

File: wiki/Help_centre/Upgrading_to_lazer/de.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+
+| Feature | stable | lazer |
+| :-- | :-- | :-- |
+| osu!taiko-Editor | ![Nein][false] | ![Ja][true] |
+| osu!catch-Editor | ![Nein][false] | ![Ja][true] |
+| osu!mania-Editor | ![Ja][true] | ![Ja][true] |
+| Öffne einen Schwierigkeitsgrad als Referenz | ![Ja][true] | ![Nein][false] |
+| SV / Lautstärke pro Objekt | ![Nein][false] | ![Ja][true] |
+| Pattern-Rotation | ![Ja][true] | ![Teilweise][partial][^editor-precise-rotation] |
+| Pattern-Skalierung | ![Nein][false] | ![Ja][true] |
```
Comment (Phippe) [inline wiki/Help_centre/Upgrading_to_lazer/de.md:89]: I guess "Muster" could be used as a translation for pattern. Or did you think that this wouldn't be clear enough to the user? Maybe you could use something like "Objektmuster" or "Notenmuster" instead if that's the case 🤔

File: wiki/Help_centre/Upgrading_to_lazer/de.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+
+#### Warum wird es "lazer" genannt?
+
+Was ist schärfer als eine scharfe Kante? Ein Laser! Es handelt sich um ein Wortspiel mit der Updatequelle `Cutting Edge`, die experimentelle Version des Clients.
+
+#### Warum dauert es so lange, bis es die "Hauptversion" wird?
+
+Auch wenn osu! wie ein einfaches Spiel aussieht, gibt es Hunderte von Funktionen und Systemen, auf die sich die Nutzer inzwischen verlassen. Je nachdem, wen man fragt, ist lazer schon seit Jahren in einem voll spielbaren Zustand oder es fehlen noch zahlreiche Funktionen.
+
+Ein weiterer Bereich, in dem große Anstrengungen unternommen wurden, ist die historische Erhaltung — es wird sichergestellt, dass die Beatmaps sich genauso verhalten wie sie es sollen, einschließlich der Randfälle, die ursprünglich nicht eingeplant waren. osu! ist ein dynamisches Ökosystem und die Nutzer haben sich die Freiheit genommen, das Spiel weit über den geplanten Umfang hinaus zu erweitern. Wir versuchen unser Bestes, um das in Zukunft zu fördern und zu unterstützen.
```
Comment (Phippe) [inline wiki/Help_centre/Upgrading_to_lazer/de.md:222]:
```suggestion
Ein weiterer Bereich, in dem große Anstrengungen unternommen wurden, ist die historische Erhaltung — es wird sichergestellt, dass die Beatmaps sich genauso verhalten, wie sie es sollen, einschließlich der Randfälle, die ursprünglich nicht eingeplant waren. osu! ist ein dynamisches Ökosystem und die Nutzer haben sich die Freiheit genommen, das Spiel weit über den geplanten Umfang hinaus zu erweitern. Wir versuchen unser Bestes, um das in Zukunft zu fördern und zu unterstützen.
```

File: wiki/Help_centre/Upgrading_to_lazer/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Help_centre/Upgrading_to_lazer/de.md:89]: `Pattern` was used before in other articles[^play-style] and is also defined in the [Duden](https://www.duden.de/rechtschreibung/Pattern), however, I can use something else.

```suggestion
| Objektmuster-Rotation | ![Ja][true] | ![Teilweise][partial][^editor-precise-rotation] |
| Objektmuster-Skalierung | ![Nein][false] | ![Ja][true] |
```

How does this look to you?

[^play-style]: Take a look at the `play style` articles for an example.

Comment (Phippe) [reply wiki/Help_centre/Upgrading_to_lazer/de.md:89]: Huh, TIL this exists in the Duden. Though, I'm not sure if this is really the correct meaning in our context (it says something about psychology and linguistics). Nevertheless, if we already use it like this somewhere else in the wiki, I'll say keep it the same for consistency's sake, even if the meaning might technically be incorrect ¯\_(ツ)_/¯

**Review by Phippe — DISMISSED** [review]
Lgtm 👍

File: wiki/Help_centre/Upgrading_to_lazer/de.md [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+| Skin-Unterstützung | ![Ja][true] | ![Teilweise][partial][^gameplay-only] |
+| Gruppierungsmodi in der Songauswahl | ![Ja][true] | ![Nein][false] |
+| Skin im Spiel / Bearbeitung des UI-Layouts | ![Nein][false] | ![Ja][true] |
+
+### Gameplay und Skinning
+
+| Feature | stable | lazer |
+| :-- | :-- | :-- |
+| Akkurate Performance-Punkte-Anzeige | ![Teilweise][partial][^online] | ![Ja][true] |
+| Mod zum Anpassen der Schwierigkeit | ![No][false] | ![Yes][true][^difficulty-adjust] |
```
Comment (The-Last-Cookie) [inline wiki/Help_centre/Upgrading_to_lazer/de.md:47]:
```suggestion
| Mod zum Anpassen der Schwierigkeit | ![Nein][false] | ![Ja][true][^difficulty-adjust] |
```

**Review by TicClick — APPROVED** [review]

**Pull request merged**